src/utils/openai.ts

```
@@ -94,7 +98,7 @@ export const generateCommitMessage = async (
 
 		return deduplicateMessages(
 			completion.choices
-				.map(choice => sanitizeMessage(choice.text!)),
+				.map(choice => sanitizeMessage(choice.message?.content ?? 'No Commit Message')),
```

We should throw an error instead of falling back to a made up commit message.